source/communication.rst

```diff
@@ -1,6 +1,17 @@
 Communication
 =============
 
+Group Web Site
+--------------
+
+We have a web site, stored in a repository at
+https://github.com/data-exp-lab/data-exp-lab.github.io/ .  You are expected to
+post news articles when you do things of note, as well as to update project
+information on that page.  All updates are conducted via pull request and git
```

Comment:
Do you want the website to always be updated with PRs? I've been committing directly to it with news items.